portfolio/src/main/webapp/index.html

```diff
@@ -37,7 +37,30 @@ <h2 id="titles">
                     <a class="button1" href="https://github.com/AbakirH">
                         <i class="fa fa-github fa-5x pad" aria-hidden="true"></i>
                     </a>
+                </div>
             </div>
         </div>
+        
+    <div id="projects">
+        <h1>Projects</h1>
+        <div id='project1' >
+            <img class='projectIMG' src="Images/pokemonGame1.png">
```

Comment:
nit: It's odd, to me, that this directory name is capitalized.

Reply:
only the title is capitalized, I am not sure what you mean?